hack/.golint_failures

```diff
@@ -498,6 +498,7 @@ staging/src/k8s.io/client-go/kubernetes/typed/core/v1
 staging/src/k8s.io/client-go/kubernetes/typed/core/v1/fake
 staging/src/k8s.io/client-go/kubernetes/typed/extensions/v1beta1/fake
 staging/src/k8s.io/client-go/kubernetes/typed/policy/v1beta1/fake
+staging/src/k8s.io/client-go/metadata/fake
```

Comment:
Can we just fix the lint errors and not add to the tech debt?

Reply:
Lint on fake is pushing the line of what I consider debt.  Just because lint suggests it doesn’t mean it really deserves the effort - in this case, godoc on fakes for documented interfaces that are consistent across the codebase and only used in tests is something that may never justify the investment.